test/unit/test-mysqlDao.js

```diff
@@ -180,6 +201,37 @@ describe('MysqlDao', function() {
         });
     });
 
+    describe('upsert', function() {
+        beforeEach(databaseSetup);
+
+        it('Should update a single row in the user_attributes table', function() {
+            return userAttrDao.upsert({
+                    user_id: 1, user_lead_id: 1, user_subscription_id: 1, site_id: 1,
```

Comment:
I know it's just a test, but I'd remove the videoblocks-domain-specific stuff from this file. This lib isn't specific to vb.